metadata-ingestion/src/datahub/ingestion/api/source_helpers.py

```diff
@@ -314,7 +314,7 @@ def auto_empty_dataset_usage_statistics(
         logger.warning(
             f"Usage statistics with unexpected timestamps, bucket_duration={config.bucket_duration}:\n"
             ", ".join(
-                str(datetime.fromtimestamp(ts, tz=timezone.utc))
+                str(datetime.fromtimestamp(ts / 1000, tz=timezone.utc))
```

Comment:
yikes - kinda bad that we missed this

Reply:
It's just in a warning that shouldn't get hit right now, but yeah, not great, because I believe this will raise an out of bounds exception